system/ee/ExpressionEngine/View/_shared/form/buttons.php

```diff
@@ -79,6 +79,7 @@
 		<button class="<?=$class?>" <?=$button['attrs']?> <?=$disabled?> name="<?=$button['name']?>" type="<?=$button['type']?>" value="<?=$button['value']?>" data-submit-text="<?=lang($button['text'])?>" data-work-text="<?=lang($button['working'])?>"><?=$button_html?><?=$button_text?></button>
 		<?php if ($i == 0 && count($submits) > 1) : ?>
 		<button type="button" class="<?=$class?> dropdown-toggle js-dropdown-toggle saving-options" data-dropdown-pos="bottom-end">
+            <span class="sr-only"><?=lang('save_btn')?></span>
```

Comment:
I don't think this label is correct here, as it's for the dropdown arrow, not for save button

Reply:
it's just label that says: 
`'save_btn' => 'Button that opens options for saving the page',`